app_dart/dev/branch_regexps.txt

```diff
@@ -0,0 +1,2 @@
+^v[0-9]+.[0-9]+.[0-9]+$
```

Comment:
dots should be preceded by backslashes, or the regexp is saying "any character"

Reply:
right!